components/Dialogs/README.md

```diff
@@ -187,6 +188,49 @@ MDCAlertAction *alertAction =
 
 ## Extensions
 
+<!-- Extracted from docs/theming.md -->
+
+### Theming
+
+You can theme an MDCDialog to match the Material Design Dialog using your app's schemes in the DialogThemer
```

Comment:
an MDCAlertController. No such thing as MDCDialog.

Reply:
Any changes should be made upstream (in the develop branch).